### PR TITLE
meinberlin/apps/dashboard: added new participatory budget to dashboard

### PR DESCRIPTION
**Pull request by sabinammm**
@Rineee 
This now contains the new module in the blueprints, but is still missing the right icon, which I think we can add later, right?

### CODE REVIEW

File: meinberlin/apps/budgeting/predicates.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import rules
+
+from adhocracy4.phases.predicates import has_feature_active
+
+
+@rules.predicate
+def phase_allows_vote(user, item):
+    if item:
+        return has_feature_active(item.module, item.__class__, 'vote')
+    return False
```
Comment (Rineee) [inline meinberlin/apps/budgeting/predicates.py:10]: Maybe move this to A4 to all other phase predicates? But on the other hand we only use that here .. @fuzzylogic2000 ?

**Comment by Rineee** [comment]
Wording and icoon is requested, but I'd say we can also change that afterwards..

File: meinberlin/apps/budgeting/phases.py [hunk 2]
```diff
@@ -55,6 +55,21 @@ class RatingPhase(phases.PhaseContent):
...
+    module_name = _('participatory budgeting 3 phases')
+
+    features = {
+        'vote': (models.Proposal,)
+    }
+
+
 phases.content.register(RequestPhase())
 phases.content.register(CollectPhase())
 phases.content.register(RatingPhase())
```
Comment (sabinammm) [inline meinberlin/apps/budgeting/phases.py:75]: @Rineee I checked it out and it looks wonderful in the dashboard. So I'm wondering, why all the other phases are registered here and the VotingPhase isn't. What shouldn't be working then?

File: meinberlin/apps/budgeting/predicates.py (hunk 1 above)
Comment (fuzzylogic2000) [reply meinberlin/apps/budgeting/predicates.py:10]: Yes, it should be moved, I think.

File: meinberlin/apps/budgeting/phases.py (hunk 2 above)
Comment (fuzzylogic2000) [reply meinberlin/apps/budgeting/phases.py:75]: Good catch. Saving the phases in the dashboard didn't work without it.

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! I will add the phase.content.register.

**Review by fuzzylogic2000 — APPROVED** [review]
Looking good!

**Pull request merged**